### DISCUSSION TITLE
No Map Received in RViz when doing the ROS2 and Gazebo Pack

### DISCUSSION THREAD

**Discussion by aasignacion** [bitcraze/discussions#1672] [Q&A] [answered]
I am following https://www.bitcraze.io/2024/09/crazyflies-adventures-with-ros-2-and-gazebo/
Same problem with the question posted at the comment section of the post.
*****
I indeed met a problem after I run “ros2 launch crazyflie_ros2_multiranger_bringup simple_mapper_simulation.launch.py”. Actually, the Gazebo and RVIZ start normally with the Crazyflie could be controlled by ROS2 teleop keyboard node.
But in RIVIZ, it shows “No map received” in Map status and “Frame [map] does not exist” Error in Global status. Then I checked topic list which output “/crazyflie/map
and /crazyflie/map_updates”. 
So I can not see the map which should be created in RVIZ. 
Checking ros2 topic echo /crazyflie/map --> no message were being published.
Here is the rqt_graph
![rosgraph](https://github.com/user-attachments/assets/fbea4145-d20f-40c1-8509-020501b1fda2)

***

Please help me. Thank you.

**Comment by knmcguire** [comment]
Could you check with the terminal in which you started the launch file, if the simple_mapper node was initialized? Or if it crashed?

**Comment by aasignacion** [comment]
The simple_mapper node was not in the ros2 node list, so probably it was not iniatilized or crashed.
Here are the screenshots of the terminal.
![Screenshot from 2024-11-28 11-27-53](https://github.com/user-attachments/assets/1068f79a-a8f2-4c00-824f-299e2c5e241c)
![Screenshot from 2024-11-28 11-27-26](https://github.com/user-attachments/assets/c2ce3720-a47e-44b4-8436-b8f0d6f14743)

**Comment by aasignacion** [comment, marked as answer]
The problem was solved by upgrading transforms3d:
pip install --upgrade transforms3d
